lib/ui/form/widget/form_survey_answer_smiley.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/theme/app_colors.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+
+const emojis = ['😡', '😕', '😐', '🙂', '😄'];
+final selectedEmojiProvider = StateProvider.autoDispose<String>(
```

Comment:
Should they be private?

```suggestion
const _emojis = ['😡', '😕', '😐', '🙂', '😄'];
final _selectedEmojiProvider = StateProvider.autoDispose<String>(
```